parts/iaasoutputs.t

```diff
@@ -0,0 +1,24 @@
+    "resourceGroup": {
+        "type": "string",
+        "value": "[variables('resourceGroup')]"
+    },
```

Comment:
From the RP standpoint outputting resource group is not needed since the RP creates the resource group.